oauthenticator/github.py

```diff
@@ -113,6 +114,19 @@ def _github_client_secret_changed(self, name, old, new):
         config=True, help="Automatically allow members of selected organizations"
     )
 
+    populate_teams_in_auth_state = Bool(
+        False,
+        help="""
+        If auth_state is enabled, add list of teams user is part of to it.
```

Comment:
*It seems it was partially re-worded?*

```suggestion
        If auth_state is enabled, add list of team users to it.
```

Reply:
I reworded for clarity